site/src/testHelpers/handlers.ts

```diff
@@ -71,12 +71,12 @@ export const handlers = [
   rest.get("/api/v2/users", async (req, res, ctx) => {
     return res(
       ctx.status(200),
-      ctx.json([M.MockUser, M.MockUser2, M.SuspendedMockUser]),
+      ctx.json({
+        users: [M.MockUser, M.MockUser2, M.SuspendedMockUser],
+        count: 26,
```

Comment:
I'm so torn about whether it's worth making this mock response nonsensical just to get a count that's high enough to test pagination. So if it's bothering you feel free to change it.

Reply:
I needed to make it pass tests since the old response was always a static `26`. I tried to provide real numbers but couldn't get things passing.

Reply:
Yeah no worries!